aspnetcore/performance/caching/memory.md

```diff
@@ -39,7 +39,7 @@ Use `System.Runtime.Caching`/`MemoryCache` as a compatibility bridge when portin
 
 * Code should always have a fallback option to fetch data and **not** depend on a cached value being available.
 * The cache uses a scarce resource, memory. Limit cache growth:
-  * Do **not** use external input as cache keys.
+  * Do **not** use external input to add cached entries. For example, don't allow user input to add a cache entry without limiting the cache size.
```

Comment:
I'm fine with your phrasing here, but perhaps this is even clearer? What do you think?

```suggestion
  * Do **not** insert external input into the cache. As an example, using arbitrary user-provided input as a cache key is not recommended since the input might consume an unpredictable amount of memory.
```